docs/faction-design.md

```diff
@@ -0,0 +1,219 @@
+
+## About
+
+The point of this document is to outline the server's current and future intended design.
+
+The current design section should be updated by contributors of pull requests which significantly affect balance or add new features. This will serve as a useful overview for new players and code contributors to understand the existing differences between the factions and a high-level overview of gameplay.
+
+The future design section should be mainly updated by maintainers as a central location for their vision for the server. This gives direction for code contributors to build their PRs on and gives players insight into what to look forward to.
+
+The main USP of TGMC is: “asymmetrical team deathmatch PvP gameplay with a high degree of QoL”.
+
+## Current Design
+
+### Visual
+
+- Xenomorphs - arachnid/dinosaur mix.
+- TG Marines - highly flamboyant armour design and accessorising similar to Destiny, or dressed in modern military attire if preferred. Ballistic, incendiary and explosive weaponry with a modern design, as well as futuristic laser weaponry.
+- Sons of Mars - orange mirrored helmets, dark orange accent colour on black armoured suits. High-tech volkite weaponry.
+
+### Gameplay
+
+The two main factions Xenomorphs and TG Marines have assymetrical combat. This means whilst there may be a few distinct exceptions in their gameplay, they each have unique strengths, weaknesses and other differences.
+
+#### Xenomorphs
+
+Xenos have 4 tiers of playable castes. Tier 1-3 have decreasing amounts of availability to players as they move up the tiers. The tiers more or less increase the combat prowess of the player with increasing tiers. They also serve to funnel some players into the weaker roles that they typically wouldn't play by having a player cap based off of total players on the team. Tier 4 allows only one playable per caste and typically comes with minimum player requirements.
+
+Xenos gain maturity over time, progressing from Young to Mature to Elder to Ancient to Primordial. At each stage their stats improve, and for some castes their abilities improve as well e.g. Widows have a greater max amount of spiderlings.
+
+The 4 main tiers are supplemented by a minion tier that consists of AI xenomorphs. This tier serves as cannon fodder to provide a low enough challenge to new and low skill marine players to learn the basics.
+
+Xenos can persistently identify themselves through a simple phrase or word, e.g. `Mature Runner (Cucumber)`, as well as through selection of a female TTS voice.
+
+**Strengths**
+- More suited to hit-and-run attacks, due to their better information gathering (X-ray and darkvision) and high movement speed (especially on weeds).
```

Comment:
Currently, marines are faster than xenos if they use drugs and cover more distance thanks to TAD, cant say that they are suited to hit-and-run attacks and tactics as a faction, but limited to certain castes like runners and hunters.